src/Ticket.php

```diff
@@ -3260,7 +3260,7 @@ public function rawSearchOptions()
             'id'                 => '32',
             'table'              => 'glpi_slalevels',
             'field'              => 'name',
-            'name'               => __('SLA') . ' ' . _n('Escalation level', 'Escalation levels', 1),
+            'name'               => sprintf('%s %s', __('SLA'), _n('Escalation level', 'Escalation levels', 1)),
```

Comment:
Hum... "SLA escalation level" seems good in english, but "SLA niveau d'escalade" sounds weird in French.

Pattern should be translated in order to fix that, but I guess this is a terrible idea for translators; it should be better to create both "SAL escalation level" and "OLA escalation level".

BTW, current fix proposal is OK for 10.0.8, new strings won't be introduced before release.

Reply:
Alternatively, since this is displayed as a sub element in the drodown, we can just keep "Escalation level"; should'nt we? This would fix all issues.